2021-10-25 08:57:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TEST" not found; SQL statement:
INSERT INTO TEST VALUES(1, 'Hello');
INSERT INTO TEST VALUES(2, 'World');
 [42102-200]
2021-10-25 08:59:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TEST" not found; SQL statement:
INSERT INTO TEST VALUES(1, 'Hello');
INSERT INTO TEST VALUES(2, 'World');
 [42102-200]
2021-10-25 09:00:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TEST" not found; SQL statement:
INSERT INTO TEST VALUES(1, 'Hello');
INSERT INTO TEST VALUES(2, 'World');
 [42102-200]
2021-10-25 09:02:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TEST" not found; SQL statement:
INSERT INTO TEST VALUES(1, 'Hello');
INSERT INTO TEST VALUES(2, 'World');
 [42102-200]
2021-10-25 09:03:13 database: wrong user or password; user: "ROOT"
org.h2.message.DbException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at com.company.Main.main(Main.java:11)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 15 more
2021-10-25 09:04:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TEST" not found; SQL statement:
INSERT INTO TEST VALUES(1, 'Hello');
INSERT INTO TEST VALUES(2, 'World');
 [42102-200]
